app/groundhog.py

```diff
@@ -94,13 +96,13 @@ def help_response():
     """
     help_message = """
         <xmp>
-           ___                           _ _                 
```

Comment:
@samwaterbury: seems that there are quite some white spaces changes. 
Can you try to let `git` ignore those?

Reply:
I will investigate

Reply:
@jayqi mentioned in https://github.com/uptake/groundhog/pull/25#discussion_r314328746 that 
> Typically, we consider not-having-trailing-whitespace to be a good best practice, and we encourage people to turn on automatic-stripping in RStudio and other text editors.

So no need to spend much time on this if it was simply removing trailing spaces. In addition, here is a great post if you indeed want to remove white space changes (which I learned from @jameslamb ): 
https://til.hashrocket.com/posts/696df00135-remove-whitespace-changes-then-git-add-p
```
git diff -w --no-color | git apply --cached --ignore-whitespace && git checkout -- . && git reset && git add -p
```

Reply:
Useful link, will definitely save that for future reference. But yeah in this case my editor just automatically stripped it so I think I'll leave it be.